src/test/java/fi/otavanopisto/kuntaapi/test/server/integrations/mikkelinyt/MikkeliNytRestTestsIT.java

```diff
@@ -283,35 +242,37 @@ private Event mockEvent(MikkeliNytMocker mocker, String kuntaApiOrganizationId,
     
     OffsetDateTime start = getOffsetDateTime(2020, 5, 6, 17, 30, TIMEZONE_ID);
     OffsetDateTime end = getOffsetDateTime(2020, 5, 6, 19, 00, TIMEZONE_ID);
-    createIdentifier(kuntaApiOrganizationId, MikkeliNytConsts.IDENTIFIER_NAME, "test-source-id", IdType.ORGANIZATION.name());
-    createIdentifier(kuntaApiEventId, MikkeliNytConsts.IDENTIFIER_NAME, eventId, IdType.EVENT.name());
-    createIdentifier(kuntaApiAttachmentId, MikkeliNytConsts.IDENTIFIER_NAME, attachmentId, IdType.ATTACHMENT.name());
+
     mocker.mockGetBinary("/" + eventThumbPath, TEST_EVENT_ATTACHMENT_TYPE, "test-image-480.jpg");
     mocker.mockGetBinary("/" + eventImagePath, TEST_EVENT_ATTACHMENT_TYPE, "test-image-1000.jpg");
 
-    return mocker.mockEvent(kuntaApiOrganizationId, kuntaApiEventId, kuntaApiAttachmentId, eventId, eventUrl, eventName, eventDescription, eventCity, eventAddres, eventZip, eventPlace, eventImage, eventThumb, start, end);
+    return mocker.mockEvent(eventId, eventUrl, eventName, eventDescription, eventCity, eventAddres, eventZip, eventPlace, eventImage, eventThumb, start, end);
   }
 
-  private void cleanMock(MikkeliNytMocker mocker, String kuntaApiOrganizationId, String kuntaApiEventId, String kuntaApiAttachmentId) {
+  private void cleanMock(MikkeliNytMocker mocker) {
     mocker.endMock();
-
-    deleteIndentifier(kuntaApiOrganizationId);
-    deleteIndentifier(kuntaApiEventId);
-    deleteIndentifier(kuntaApiAttachmentId);
+  }
+  
+  private void createSystemSettings() {
+    insertSystemSetting(PtvConsts.SYSTEM_SETTING_BASEURL, String.format("%s%s", getWireMockBasePath(), BASE_URL));
   }
 
-  private void createSettings(String kuntaApiOrganizationId, String baseUrl, String imagesBasePath) {
-    insertOrganizationSetting(kuntaApiOrganizationId, MikkeliNytConsts.ORGANIZATION_SETTING_APIKEY, ORGANIZATION_SETTING_APIKEY);
-    insertOrganizationSetting(kuntaApiOrganizationId, MikkeliNytConsts.ORGANIZATION_SETTING_LOCATION, ORGANIZATION_SETTING_LOCATION);
-    insertOrganizationSetting(kuntaApiOrganizationId, MikkeliNytConsts.ORGANIZATION_SETTING_BASEURL, baseUrl);
-    insertOrganizationSetting(kuntaApiOrganizationId, MikkeliNytConsts.ORGANIZATION_SETTING_IMAGEBASEURL, String.format("%s/%s/1000/", baseUrl, imagesBasePath));
+  private void createOrganizationSettings(String organizationId, String baseUrl, String imagesBasePath) {
+    insertOrganizationSetting(organizationId, MikkeliNytConsts.ORGANIZATION_SETTING_APIKEY, ORGANIZATION_SETTING_APIKEY);
+    insertOrganizationSetting(organizationId, MikkeliNytConsts.ORGANIZATION_SETTING_LOCATION, ORGANIZATION_SETTING_LOCATION);
+    insertOrganizationSetting(organizationId, MikkeliNytConsts.ORGANIZATION_SETTING_BASEURL, baseUrl);
+    insertOrganizationSetting(organizationId, MikkeliNytConsts.ORGANIZATION_SETTING_IMAGEBASEURL, String.format("%s/%s/1000/", baseUrl, imagesBasePath));
+  }
+  
+  private void deleteOrganizationSettings(String organizationId) {
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Remove this unused private "deleteOrganizationSettings" method. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.com/coding_rules#rule_key=squid%3AUnusedPrivateMethod)